app/grandchallenge/reader_studies/views.py

```diff
@@ -354,13 +354,25 @@ def get_context_data(self, **kwargs):
 
 
 class ReaderStudyLeaderBoard(
-    LoginRequiredMixin, ObjectPermissionRequiredMixin, DetailView
+    LoginRequiredMixin, UserPassesTestMixin, DetailView
```

Comment:
I'm not 100% sure anymore what the correct way is. As Thomas found out, using `AccessMixin` together with `LoginRequiredMixin` doesn't work unless you create a seperate class for the permission check. `UserPassesTestMixin` can be used like this directly, but can't be combined with other UserPassesTestMixins - not a problem now, per se, but could be in the future. The choice depends a bit on what we decide to do in https://github.com/comic/grand-challenge.org/pull/3928

Reply:
No longer relevant thanks to James' suggestion of using a custom permission.